gradle/gradle-publish.gradle

```diff
@@ -0,0 +1,77 @@
+apply plugin: 'maven-publish'
+apply plugin: 'signing'
+
+
+ext["signing.keyId"] = System.getenv('SIGNING_KEY_ID') ?: ''
+ext["signing.password"] = System.getenv('SIGNING_KEY_PASSWORD') ?: ''
+ext["signing.secretKeyRingFile"] = System.getenv('SIGNING_KEY_FILE') ?: ''
+ext["SONATYPE_NEXUS_USERNAME"] = System.getenv('SONATYPE_NEXUS_USERNAME') ?: ''
+ext["SONATYPE_NEXUS_PASSWORD"] = System.getenv('SONATYPE_NEXUS_PASSWORD') ?: ''
+
+
+afterEvaluate {
+    android.libraryVariants.each { variant ->
+        publishing.publications.create variant.name, MavenPublication, {
+            groupId group
+            version project.version
+            artifactId project.name.toLowerCase()
+            from components."${variant.name}"
+            artifact androidSourcesJar
```

Comment:
Not sure exactly the kotlin equivalent of javadocs but we are missing publishing those here - Should we add a TODO somewhere so we don't forget that before release?

Reply:
im good with adding a TODO, im not really sure how to generate those. I dont want to just paste something from `stackoverflow`. Maybe we can also add a ticket and sync later on it. Thoughts?

Reply:
Yeah let's just create a JIRA card for that